api/__init__.py

```diff
@@ -1,39 +1 @@
-"""
-Atmosphere service utils for rest api.
-
-"""
-from functools import wraps
-import os.path
-import uuid
-
-from django.core.exceptions import ObjectDoesNotExist
-
-from rest_framework.response import Response
-
-from threepio import logger, api_logger
-
-import rtwo.compute  # Necessary to initialize Meta classes
-
-from core.models import AtmosphereUser as User
-
-
-def emulate_user(func):
-    """
-    Support for staff users to emulate a specific user history.
-
-    This decorator is specifically for use with an APIView.
-    """
-    @wraps(func)
-    def wrapper(self, *args, **kwargs):
-        emulate_name = self.request.query_params.get('username', None)
-        if self.request.user.is_staff and emulate_name:
-            emualate_name = emulate_name[0]  # Querystring conversion
-            original_user = self.request.user
-            try:
-                self.request.user = User.objects.get(username=emulate_name)
-                self.emulated = (True, original_user, emulate_name)
-            except User.DoesNotExist:
-                self.emulated = (False, original_user, emulate_name)
-        return func(self, *args, **kwargs)
-    return wrapper
```

Comment:
This removal was part of commit d8df468f77278b1ea5867db15861d91ff1edb968 - but the message doesn't give a reason for why it was removed. 

I notice this because I am unable to do an `emulate_user` from DRF view ([ATMO-1323](https://pods.iplantcollaborative.org/jira/browse/ATMO-1323))